docs/Security.md

```diff
@@ -0,0 +1,187 @@
+
+Splunk Enterprise provides a range of security frameworks that are also available in your Kubernetes deployment.  Before moving forward take the time to familiarize yourself with these frameworks here: [Secure Splunk](https://docs.splunk.com/Documentation/Splunk/8.1.2/Security/WhatyoucansecurewithSplunk)
+
+Some deployments require communication with instances outside the Kubernetes cluster. For example, a Forwarders outside the Kubernetes cluster ingesting data into a Indexer Cluster inside Kubernetes. While there are many more possible scenarios, we only support access from outside the Kubernetes cluster for Ingesting data (Forwarders), access to Splunk Web and Rest APIs.
+
+**These are the scenarios currently supported.**
+
+|Outside Kubernetes   | SSL/TLS Configuration | Supported |
+|---|---|---|
+|Forwarders   | Gateway Termination | YES   |
+|Forwarders   | End-to-End Termination | YES   |
+|Splunk Web   | End-to-End Termination | YES  |
+|REST API   | End-to-End Termination | YES  |
+|Splunk Web   | Gateway Termination | NO  |
+|REST API   | Gateway Termination | NO  |
+
+
+
+
+## Securing Splunk with TLS
+
+We recommend you use SSL/TLS certificates to secure communication between instances. In Kubernetes we use the same principles defined here: [Secure Splunk with SSL](https://docs.splunk.com/Documentation/Splunk/8.1.2/Security/AboutsecuringyourSplunkconfigurationwithSSL)
+
+For information on how to create your own certificates or how to sign third-party certificates please follow the steps in this documentation: [How to Sign Certificates](https://docs.splunk.com/Documentation/Splunk/8.1.2/Security/Howtoself-signcertificates)
+
+In kubernetes we will apply the certificates as well as the configuration files using an app. Follow the steps in this documentation to [Install Apps using Splunk Operator](https://github.com/splunk/splunk-operator/blob/develop/docs/Examples.md#installing-splunk-apps).
+
+
+
+## Securing Splunk Web
+
+Once you have your certificates ready we need to configure web.conf to enable encryption on Splunk Web. Follow the steps here [Secure Splunk Web with TLS](https://docs.splunk.com/Documentation/Splunk/8.1.2/Security/SecureSplunkWebusingasignedcertificate) to create your configuration file.
+
+Next we will create an app with both certificates and the web.conf configured. This is an example of a sample app called "myapp" with such configuration.
+
+```
+myapp
+├── default
+│   ├── app.conf
+│   └── web.conf
+└── mycerts
+    ├── mySplunkCertificate.pem
+    └── mySplunkPrivateKey.key
+```
+
+Sample web.conf
+```
+[settings]
+enableSplunkWebSSL = true
+privKeyPath = $SPLUNK_HOME/etc/apps/myapp/mycerts/mySplunkPrivateKey.key
+serverCert =  $SPLUNK_HOME/etc/apps/myapp/mycerts/mySplunkCertificate.pem
+```
+
```

Comment:
We have explained what an app should contain, but not how the app should be deployed, & where. Same in other places too